scene/gui/line_edit.cpp

```diff
@@ -2052,12 +2093,14 @@ Size2 LineEdit::get_minimum_size() const {
 	// Take icons into account.
 	int icon_max_width = 0;
 	if (right_icon.is_valid()) {
-		min_size.height = MAX(min_size.height, right_icon->get_height());
-		icon_max_width = right_icon->get_width();
+		Point2 right_icon_size = _get_right_icon_size(right_icon);
+		min_size.height = MAX(min_size.height, right_icon_size.height);
+		icon_max_width = right_icon_size.width;
 	}
 	if (clear_button_enabled) {
-		min_size.height = MAX(min_size.height, theme_cache.clear_icon->get_height());
-		icon_max_width = MAX(icon_max_width, theme_cache.clear_icon->get_width());
+		Point2 right_icon_size = _get_right_icon_size(theme_cache.clear_icon);
+		min_size.height = MAX(min_size.height, right_icon_size.height);
+		icon_max_width = MAX(icon_max_width, right_icon_size.width);
```

Comment:
If right icon is bigger, the minimum size will fit to the right icon, even if it's not used when clear button is enabled.
It's a pre-existing bug, so you don't need to fix it now.